src/main/java/net/dries007/tfc/objects/blocks/agriculture/BlockFruitTreeTrunk.java

```diff
@@ -261,6 +285,17 @@ public void randomTick(World worldIn, BlockPos pos, IBlockState state, Random ra
         }
     }
 
+    @Override
+    @SuppressWarnings("deprecation")
+    public void neighborChanged(IBlockState state, World worldIn, BlockPos pos, Block blockIn, BlockPos fromPos)
+    {
+        super.neighborChanged(state, worldIn, pos, blockIn, fromPos);
+        if (!BlocksTFC.isGrowableSoil(worldIn.getBlockState(pos.down())))
+        {
+            worldIn.destroyBlock(pos, false);
+        }
```

Comment:
Does this not break any trunks that are > 1 high, rendering fruit trees impossible to grow?

Reply:
you're right! I should've added another condition for trunks!

Reply:
This is still broken - fruit tree branches currently break on every block update.